tensorflow/contrib/tensorrt/resources/TRTResources.h

```diff
@@ -0,0 +1,91 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_CONTRIB_TENSORRT_RESOURCES_TRTRESOURCES_H_
+
+#define TENSORFLOW_CONTRIB_TENSORRT_RESOURCES_TRTRESOURCES_H_
+
+#include <list>
+#include <sstream>
+#include <string>
+#include <thread>
+#include <vector>
+#include "tensorflow/contrib/tensorrt/log/trt_logger.h"
+#include "tensorflow/contrib/tensorrt/resources/TRTInt8Calibrator.h"
+#include "tensorflow/core/framework/resource_mgr.h"
+#include "tensorrt/include/NvInfer.h"
+
+namespace tensorflow {
+namespace trt {
+struct TRTCalibrationResource : public tensorflow::ResourceBase {
```

Comment:
Since the base class is a 'class', use 'class' here instead of 'struct'? Same below for other structs.

Reply:
That was how given in the example of resource manager https://github.com/tensorflow/tensorflow/blob/b07680459a88224fce83daa7b3b70bcc62b9c896/tensorflow/core/framework/resource_mgr.h#L55

Reply:
Fair enough. I think the comment in the header is stale, as most code uses 'class'. But never mind in this case.

Reply:
Changed them to class nevertheless